tests/periph_flashpage/main.c

```diff
@@ -27,25 +27,22 @@
 
 #define LINE_LEN            (16)
 
-/**
- * @brief   Allocate space for 1 flash page in RAM
- */
-static uint8_t page_mem[FLASHPAGE_SIZE];
+#ifndef MODULE_PERIPH_FLASHPAGE_RAW
+#define ALIGNMENT_OPTS
+#else
+#define ALIGNMENT_OPTS __attribute__ ((aligned (FLASHPAGE_RAW_ALIGNMENT)))
 
-#ifdef MODULE_PERIPH_FLASHPAGE_RAW
 /*
  * @brief   Allocate an aligned buffer for raw writings
  */
-static char raw_buf[64] __attribute__ ((aligned (FLASHPAGE_RAW_ALIGNMENT)));
-
-static uint32_t getaddr(const char *str)
-{
-    uint32_t addr = strtol(str, NULL, 16);
-
-    return addr;
-}
+static char raw_buf[64] ALIGNMENT_OPTS;
 #endif
 
+/**
+ * @brief   Allocate space for 1 flash page in RAM
+ */
+static uint8_t page_mem[FLASHPAGE_SIZE] ALIGNMENT_OPTS;
```

Comment:
Did you need this because stm32l4 requires 64 alignment ?

Should it not be always aligned to specific size and not only when there is `MODULE_PERIPH_FLASHPAGE_RAW` ? I say this as it is casted to `uint32_t/uint64_t` in some implementation.

If that is the case, it is a previously existing problem and having this alignment defined all the time would be another PR but at least putting a comment somewhere or in the commit message would be great.